modules/ROOT/pages/https-reference.adoc

```diff
@@ -2,9 +2,33 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: configure, HTTPS endpoint
+
+You can automatically deploy an API proxy using an HTTP listener (inbound HTTP endpoint) with minimal setup. You can use this approach if you do not foresee any security concerns in your Anypoint Platform API setup. Otherwise, you must configure your API proxy with an HTTPs listener (inbound HTTPS endpoint), instead of an HTTP listener.
+
+MuleSoft requires that you provide your own certificate when configuring your HTTPs setup. The certificate you use can be a self-signed certificate to be used for demo purposes, custom TLS certificate with validity within your organization, or a trusted Certificate Authority (CA) certificate.
+
+If your backend uses HTTPs with a trusted CA certificate, you do not need to perform any special action to communicate with it from the API proxy. If your backend uses HTTPs with a self-signed certificate or custom TLS certificate, you must configure the API proxy requester with the corresponding certificate.
+
+Depending on your business use case, you can use any of the following recommended ways to configure your API proxy to use a certificate when setting up an HTTPS listener:
+
+* xref:building-https-proxy.adoc[Certificate configured through the UI by leveraging Secret Manager]:
++ 
+The certificate is uploaded to a secure vault within API Platform, and then made available when configuring the proxy through the Secret Manager. Once configured, you can deploy the secure API proxy to the target runtime, such as CloudHub, Runtime Fabric, and on-premises.
+* xref:proxy-domain-support.adoc[Certificate defined in already deployed Mule Domain]
++
+The certificate is configured in the domain and then the API proxy HTTPS listener references it. Once configured, you can deploy the secure API pProxy to the target runtime, such as CloudHub, Runtime Fabric, and on-premises. CloudHub is not supported with this approach because,because as it does not support Mule domains.
```

Comment:
ditto previous edit
```suggestion
Configure the certificate in the domain so that the API proxy HTTPS Listener references it, and then deploy the secure API proxy to the target Runtime Fabric, or on-premises target. (CloudHub is not supported with this approach because it does not support Mule domains.)
```

Reply:
Done! Thanks!